src/explore-education-statistics-admin/src/pages/methodology/edit-methodology/status/MethodologyStatusPage.tsx

```diff
@@ -28,101 +27,92 @@ const statusMap: Dictionary<string> = {
   Approved: 'Approved',
 };
 
-const MethodologyStatusPage = ({
-  match,
-}: RouteComponentProps<MethodologyRouteParams>) => {
-  const { methodologyId } = match.params;
+const MethodologyStatusPage = () => {
+  const {
+    methodologyId,
+    methodology: currentMethodology,
+    onMethodologyChange,
+  } = useMethodologyContext();
 
   const [isEditing, toggleForm] = useToggle(false);
 
-  const {
-    value: model,
-    setState: setModel,
-    isLoading,
-  } = useAsyncRetry(async () => {
-    const [summary, canApprove, canMarkAsDraft] = await Promise.all([
-      methodologyService.getMethodology(methodologyId),
+  const { value: permissions, isLoading } = useAsyncRetry(async () => {
+    const [canApprove, canMarkAsDraft] = await Promise.all([
       permissionService.canApproveMethodology(methodologyId),
       permissionService.canMarkMethodologyAsDraft(methodologyId),
     ]);
 
     return {
-      summary,
       canApprove,
       canMarkAsDraft,
     };
   }, [methodologyId]);
 
   const handleSubmit = async (values: FormValues) => {
-    if (!model) {
+    if (!currentMethodology) {
       return;
     }
 
     const nextSummary = await methodologyService.updateMethodology(
       methodologyId,
       {
-        ...model.summary,
+        ...currentMethodology,
         ...values,
```

Comment:
I suspect this is going to cause a merge conflict with Ben's imminent PR: https://github.com/dfe-analytical-services/explore-education-statistics/pull/2785